src/emissions-workspace/emissions.module.ts

```diff
@@ -13,21 +13,25 @@ import { EmissionsChecksService } from './emissions-checks.service';
 import { WeeklyTestSummaryCheckService } from '../weekly-test-summary-workspace/weekly-test-summary-check.service';
 import { PlantRepository } from '../plant/plant.repository';
 import { HourlyOperatingWorkspaceModule } from '../hourly-operating-workspace/hourly-operating.module';
+import { MonitorLocationWorkspaceModule } from '../monitor-location-workspace/monitor-location.module';
+import { WeeklyTestSummaryWorkspaceModule } from '../weekly-test-summary-workspace/weekly-test-summary.module';
 
 @Module({
   imports: [
     TypeOrmModule.forFeature([EmissionsWorkspaceRepository]),
     TypeOrmModule.forFeature([PlantRepository]),
     DailyTestSummaryWorkspaceModule,
     HourlyOperatingWorkspaceModule,
+    MonitorLocationWorkspaceModule,
+    WeeklyTestSummaryWorkspaceModule,
   ],
   controllers: [EmissionsWorkspaceController],
   providers: [
     EmissionsMap,
     EmissionsWorkspaceService,
     EmissionsSubmissionsProgressMap,
     EmissionsChecksService,
-    WeeklyTestSummaryCheckService,
+    // WeeklyTestSummaryCheckService,
```

Comment:
If this isn't needed it should be removed along with it's import.

Reply:
Ah woops should be removed now.